<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>833</width>
    <height>457</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
font-size:24px;
background:rgb(163, 96, 240);
}

QLabel
{
font-size: 30px;
}

#errorMessageLabel
{
background:transparent;
color: rgb(255, 0, 0);
font-size: 16px;
}

QFrame
{
background:#333;
border-radius: 15px;
}

QPushButton
{
border: 1px rgb(255, 141, 144);
background:rgb(255, 141, 144);
border-radius:9px;
color: #333;
}
QPushButton:hover:!pressed
{
background:rgb(204, 114, 117);
border-width:2px;
border-color: white;
border-style:solid;
border-radius:9px;
color: #333;
}
#playButton
{
 border-radius:50px;
 border-color: red;
 max-width:200px;
 max-height:100px;
 min-width:400px;
 min-height:100px;
}

QToolButton
{
 background-color: rgb(255, 141, 144);
 border-width:1px;
 border-radius:50px;
 border-color: red;
 max-width:100px;
 max-height:100px;
 min-width:100px;
 min-height:100px;
}

QLineEdit
{
background:transparent;
border:none;
color:#717072;
}

QTextEdit
{
background:transparent;
border:none;
color:#717072;
}</string>
  </property>
  <widget class="QFrame" name="usernameFrame">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>241</width>
     <height>141</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="usernameLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>username</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QToolButton" name="userIcon">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/images/ice-cream.png</normaloff>:/images/images/ice-cream.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="highScoresButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>90</y>
     <width>401</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Resent Scores</string>
   </property>
  </widget>
  <widget class="QPushButton" name="playButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>270</y>
     <width>400</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string>Play</string>
   </property>
  </widget>
  <widget class="QFrame" name="scoreDateFrame">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>321</width>
     <height>221</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QTextEdit" name="scoreEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>301</width>
      <height>101</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="markdown">
     <string/>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.AppleSystemUIFont'; font-size:24px; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="dateLineEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Date</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>High Score: 700</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QLineEdit" name="birthdayLineEdit">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>170</y>
     <width>361</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
color: yellow;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
